src/multiplex/client.rs

```diff
@@ -140,19 +215,22 @@ where
 
 // ===== ClientInner =====
 
-struct Pending<Tag, Item> {
+pub struct Pending<Tag, Res> {
     tag: Tag,
-    tx: tokio::sync::oneshot::Sender<ClientResponse<Item>>,
+    tx: tokio::sync::oneshot::Sender<ClientResponse<Res>>,
```

Comment:
Here's another thing we need to document — if a `Pending` is dropped, what happens to the caller that made the dropped request? I don't actually remember what we do if the corresponding `Receiver.await` encounters a closed channel.